src/types/commands.ts

```diff
@@ -196,6 +198,9 @@ export const coreTypes = new Set<CoreCommandTypes>([
   /** FILTERS */
   "CREATE_FILTER_TABLE",
   "REMOVE_FILTER_TABLE",
+
+  /** IMAGE */
+  "CREATE_FIGURE_IMAGE",
```

Comment:
for consistency with the other command, we could simply rename it CREATE_IMAGE ? after discussing it internally, we won't need another dedicated command to create images (for instance inside a cell)